MultiVersions/Fabric/main/numeralping/lang/zh_tw.json

```
@@ -0,0 +1,23 @@
+{
+  "numeralping.option.playerList": "玩家清單",
+  "numeralping.option.playerList.desc": "在玩家清單中啟用數字延遲顯示。",
+  "numeralping.option.serverList": "伺服器清單",
+  "numeralping.option.serverList.desc": "在伺服器清單中啟用數字延遲顯示。",
```

在大多數遊戲中，玩家應該都習慣叫 `Ping 值`，而非 `延遲顯示`
```suggestion
  "numeralping.option.serverList.desc": "在伺服器清單中啟用數值化 Ping 值。",
```